src/main/java/dataMgmt/models/FileMetadata.java

```diff
@@ -0,0 +1,63 @@
+package dataMgmt.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Created by fahslaj on 5/2/2016.
+ */
+public class FileMetadata {
+
+    @JsonProperty("FileID")
+    protected long fileId;
+
+    @JsonProperty("FilePath")
+    protected String filePath;
+
+    @JsonProperty("Version")
+    protected long version;
```

Comment:
A thought about storing version: shouldn't we mainly handle version number storage in session data rather than writing it to the disk on every new version? The version should only be written to the disk when the project is disconnected from the server and no longer receiving updates.